src/test/java/com/rednavis/vaadin/showcase/db/PostgreSqlDbTest.java

```diff
@@ -0,0 +1,50 @@
+package com.rednavis.vaadin.showcase.db;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.rednavis.vaadin.showcase.rule.testcontainer.PostgreSqlDbRule;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class PostgreSqlDbTest {
+
+  private static final String TABLE_NAME = "test";
+  private static final String COLUMN_ID = "id";
+  private static final String COLUMN_NAME = "name";
+  
+  @Rule
+  public PostgreSqlDbRule dbRule = new PostgreSqlDbRule();
+
+  /**
+   * Initialization before test.
+   */
+  @Before
+  public void init() throws SQLException {
+    Map<String, String> columns = new HashMap<>();
+    columns.put(COLUMN_ID, "int");
+    columns.put(COLUMN_NAME, "varchar(255)");
+    dbRule.createTable(TABLE_NAME, columns);
+  }
+
+  @After
+  public void destroy() throws SQLException {
+    dbRule.dropTable(TABLE_NAME);
+  }
+
+  @Test
+  public void test_successfully_write_in_db() throws SQLException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The JUnit 4 test method name 'test_successfully_write_in_db' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4816225)